src/test/java/com/virtuslab/archunit/ClassStructureTestSuite.java

```diff
@@ -18,6 +22,37 @@ public void actions_implementing_DumbAware_should_extend_DumbAwareAction() {
         .check(importedClasses);
   }
 
+  @Test
+  public void actions_overriding_onUpdate_should_call_super_onUpdate() {
+    classes()
+        .that()
+        .areAssignableTo(com.virtuslab.gitmachete.frontend.actions.base.BaseProjectDependentAction.class)
+        .and()
+        .areNotAssignableFrom(com.virtuslab.gitmachete.frontend.actions.base.BaseProjectDependentAction.class)
```

Comment:
Basically, **subclasses** of `BaseProjectDependentAction`... I can't see any easier way to express that, only with these upper/lower bounds criteria